app/assets/javascripts/jsxc.js

```diff
@@ -42,6 +42,12 @@ $(document).ready(function() {
             onlogin: true
           }
         });
+
+        // auto login if the user hasn't disabled it
+        var onLogin = !(app.currentUser.get("disable_chat_login"));
+        if (onLogin) {
+          jsxc.gui.changePresence("online");
+        }
```

Comment:
I think `autoLogin` would be a better name for the variable. Or just drop the variable since it is only used in the next line.